<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-N60QB27" ignored="0" name="Test" tests="3" failures="2" timestamp="2022-11-19T00:07:46 IST" time="4.306" errors="0">
  <testcase name="getCustomerAPIWithValidSecretKey" time="3.459" classname="testcases.CreateCustomerTest"/>
  <testcase name="validateCreateCustomerAPIWithValidSecretKey" time="0.773" classname="testcases.CreateCustomerTest2">
    <failure type="java.lang.AssertionError" message="Status code is not as expected.Hence Test Failed. expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: Status code is not as expected.Hence Test Failed. expected [200] but found [401]
at testcases.CreateCustomerTest2.validateCreateCustomerAPIWithValidSecretKey(CreateCustomerTest2.java:23)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- validateCreateCustomerAPIWithValidSecretKey -->
  <testcase name="invalidateCreateCustomerAPIWithValidSecretKey" time="0.014" classname="testcases.CreateCustomerTest2">
    <failure type="java.lang.IllegalArgumentException" message="userName cannot be null">
      <![CDATA[java.lang.IllegalArgumentException: userName cannot be null
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:86)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:58)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.common.assertion.AssertParameter.notNull(AssertParameter.groovy:26)
at io.restassured.internal.common.assertion.AssertParameter$notNull.callStatic(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallStatic(CallSiteArray.java:54)
at io.restassured.internal.common.assertion.AssertParameter$notNull.callStatic(Unknown Source)
at io.restassured.internal.AuthenticationSpecificationImpl.basic(AuthenticationSpecificationImpl.groovy:46)
at testcases.CreateCustomerTest2.invalidateCreateCustomerAPIWithValidSecretKey(CreateCustomerTest2.java:33)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- invalidateCreateCustomerAPIWithValidSecretKey -->
</testsuite> <!-- Test -->
